src/components/ErrorBoundary/index.tsx

```diff
@@ -1,66 +1,157 @@
 import { Trans } from '@lingui/macro'
 import * as Sentry from '@sentry/react'
-import React, { PropsWithChildren } from 'react'
+import { ButtonLight, ButtonPrimary } from 'components/Button'
+import { ChevronUpIcon } from 'nft/components/icons'
+import { useIsMobile } from 'nft/hooks'
+import React, { PropsWithChildren, useState } from 'react'
+import { Copy } from 'react-feather'
 import styled from 'styled-components/macro'
 
-import { ExternalLink, ThemedText } from '../../theme'
-import { AutoColumn } from '../Column'
-import { AutoRow } from '../Row'
+import { CopyToClipboard, ExternalLink, ThemedText } from '../../theme'
+import { Column } from '../Column'
 
 const FallbackWrapper = styled.div`
   display: flex;
-  flex-direction: column;
-  width: 100%;
-  align-items: center;
-  z-index: 1;
+  width: 100vw;
+  height: 100vh;
 `
 
 const BodyWrapper = styled.div<{ margin?: string }>`
+  width: min(100%, 500px);
```

Comment:
why not just min-width

Reply:
Actually, I think this could be `max-width: 500px` and `width: 100%`! `min-width` would make it render full screen on all sizes

Reply:
right